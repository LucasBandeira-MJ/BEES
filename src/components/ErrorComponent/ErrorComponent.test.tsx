import renderer from 'react-test-renderer'

import { ErrorComponent } from '.'

describe("ErrorComponent component", () => {
    it("should match snapshot", () => {
        const snapshot = renderer
        .create(<ErrorComponent />)
        .toJSON()

        expect(snapshot).toMatchSnapshot()
    })
})