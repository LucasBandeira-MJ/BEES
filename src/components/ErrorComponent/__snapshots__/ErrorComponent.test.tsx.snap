// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorComponent component should match snapshot 1`] = `
<div
  className="sc-bczRLJ bCbEtg"
>
  <h1>
    Oops, something went wrong...
  </h1>
  <p>
    Please try again later
  </p>
  <img
    alt="Spilled beer"
    src="images/spilledbeer.svg"
  />
</div>
`;
